### ISSUE TITLE
Invoking ISftpClient.Dispose() results in [CS0121] compile error

### ISSUE THREAD

**Issue by hthroweriii** [sshnet/SSH.NET#1239]
**Version: 2023.0.0**

### Problem ###
Directly invoking ISftpClient.Dispose() results in the following error: 

> [CS0121] The call is ambiguous between the following methods or properties: 'Renci.SshNet.IBaseClient.Dispose()' and 'System.IDisposable.Dispose()'

The following displays sample code to reproduce the compile time error.
```
ISftpClient sftpClient = new SftpClient(host, port, username, password);
sftpClient.Dispose(); //This line results in the [CS0121] error
```

### Current Workaround ###

```
ISftpClient sftpClient = new SftpClient(host, port, username, password);
((IDisposable)sftpClient).Dispose(); 
```

**Comment by Rob-Hague** [comment]
Thanks for raising. This was fixed by #1229 which will be in the next version.

**Comment by WojciechNagorski** [comment]
The 2023.0.1 version has been released to Nuget: https://www.nuget.org/packages/SSH.NET/2023.0.1
